classes/plugins/ThemePlugin.inc.php

```diff
@@ -821,6 +824,57 @@ function isColourDark( $colour, $limit = 130 ) {
 		);
 		return $contrast < $limit;
 	}
+	
+	/**
+	 * Add default stylesheets to the HTML galley
+	 *
+	 * Passes styles to the galley through HTML Galley Plugin hook
+	 *
+	 * @param $hookName string
+	 * @param $args array [
+	 * $args[0] ArticleGalley
+	 * $args[1] string HTML
+	 * ]
+	 *
+	 * @return bool
+	 */
+	function addHmtlGalleyStyle($hookName, $args) {
+		$galley =& $args[0];
+		$contents =& $args[1];
+		
+		if (empty($contents)) return false;
+		
+		$submissionFile = $galley->getFile();
+		
+		// Check if HTML galley has attached styles
+		$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
+		import('lib.pkp.classes.submission.SubmissionFile');
+		$embeddableFiles = array_merge(
+			$submissionFileDao->getLatestRevisions($submissionFile->getSubmissionId(), SUBMISSION_FILE_PROOF),
+			$submissionFileDao->getLatestRevisionsByAssocId(ASSOC_TYPE_SUBMISSION_FILE, $submissionFile->getFileId(), $submissionFile->getSubmissionId(), SUBMISSION_FILE_DEPENDENT)
+		);
```

Comment:
If you add the `$embeddableFiles` to the hook in the OJS commit, you can avoid these requests to the DB.